pyaedt/desktop.py

```diff
@@ -691,6 +624,113 @@ def disable_autosave(self):
         """
         self._main.oDesktop.EnableAutoSave(False)
 
+    def change_license_type(self, license_type="Pool"):
+        """Change the License Type between ``Pack`` and ``Pool``.
+
+        ..note: The command returns True even if the Key is wrong due to API limitation.
+
+        Parameters
+        ----------
+        license_type : str, optional
+            Set License Type between ``Pack`` and ``Pool``.
+
+        Returns
+        -------
+        bool
+        """
+        try:
+            self._main.oDesktop.SetRegistryString("Desktop/Settings/ProjectOptions/HPCLicenseType", license_type)
+            return True
+        except:
```

Comment:
We should log failure here.

Reply:
the API doesn't return always error unfortunately